views/partials/head.html

```diff
@@ -6,6 +6,6 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="canonical" href="https://userstyles.world">
 <link rel="stylesheet" href="/css/main.css" type="text/css" media="screen">
-<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+<link rel="mask-icon" href="/gopher-mascot.svg" color="blue">
```

Comment:
Hmm, well I forgot about this one. We might have to restore the previous mascot SVG just for this purpose.

Reply:
maybe, it's purely for IOS users an icon to have on their home screen when you add userstyles.world to it.